plugin.video.regiotv/resources/lib/data.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+''' A list of static data '''
+
+from __future__ import absolute_import, division, unicode_literals
+
+CHANNELS = [
```

Comment:
Are all these URL's freely available? And if so, where?

Reply:
This was asked in the previous review as well. The URLs are available from each website.